app/services/user.js

```diff
@@ -99,3 +101,29 @@ exports.getAlbumsOfUser = async (userEmail, userId, admin, getAlbum) => {
     throw errors.defaultError(e.message);
   }
 };
+exports.getPhotosOfAlbums = async (userEmail, albumId, getPhotosOfAlbum) => {
+  try {
+    const user = await User.findOne({ where: { email: userEmail } });
+    if (!user) {
+      throw errors.badRequest('User not exist');
+    }
+    const transaction = await AlbumTransaction.findOne({ where: { userId: user.id, albumId } });
+    if (!transaction) {
+      throw errors.badRequest('You did not buy this album');
+    }
+    return getPhotosOfAlbum(albumId);
+  } catch (e) {
+    if (e.internalCode) {
+      throw e;
+    }
+    throw errors.defaultError(e.message);
+  }
+};
+
+exports.invalidateAllTokens = async () => {
+  try {
+    return await User.update({ timestampTokenCreation: moment().format() }, { where: {} });
```

Comment:
if you are using only a promise, return only this promise. Not use try and catch in this case